### DISCUSSION TITLE
How to override the toGlyph function?

### DISCUSSION THREAD

**Discussion by rica-carv** [e107inc/e107#5301] [Q&A Core]
How is it possible to override the toGlyph function at theme level?

Imagine i don't want to use Fontawesom, but another free icons with SVG. Naturally they have different names and since the toGlyph is the main bridge between the icons font and the frontend, it must be changed to aid in icons font change.

Or there's no other option that to use Fontawesome? No custom icons? Or do i have to change and put in theme any single template that use the toGlyph function?

**Comment by Jimmi08** [comment]
Yes, there was a way...  in theme xml define your own icon files.  But it worked only for use in TinyMCE (selecting icons).   If you need this, I could find a theme where this was used.  It was a bit tricky to get it loaded. 

I think that old constants still work... not sure.    

My problem with icon picker was that it offered not supported icon sets.  

I don't remember more.  Maybe there is other way.  But example what you need exactly would be nice.
